src/strings/ara.json

```diff
@@ -51,6 +51,7 @@
     "Help_Improve": "ساعد(ي) في تحسين أبرتيُم!",
     "Contact_Us": "نرحب بتواصلك معنا في حالة العثور على خطأ، أو إن كان هناك مشروع تود(ين) أن تطرح(ي) علينا العمل عليه، أو إن أردتم المساعدة.",
     "Maintainer": "هذا الموقع يتكفل بصيانته {{maintainer}}.",
+    "More_Languages": "تبحث عن المزيد من اللغات؟ جرب {{more_languages}}",
```

Comment:
Do you know all of these languages? If not, you shouldn't be adding translations for all of them.

Reply:
Ayo, my bad on that one. I didn't realize I shouldn't be adding translations for languages I'm not familiar with. Should've checked with you before making those changes. Thanks for pointing it out! I've reverted the changes as requested.